sdk/core/System.BinaryData/src/BinaryData.cs

```diff
@@ -262,16 +263,21 @@ public T ToObjectFromJson<T>(JsonSerializerOptions? options = default)
         /// <param name="data">The value to be converted.</param>
         public static implicit operator ReadOnlyMemory<byte>(
             BinaryData data) =>
-            data._bytes;
+            data?._bytes ?? default;
 
         /// <summary>
         /// Defines an implicit conversion from a <see cref="BinaryData" /> to a <see cref="ReadOnlySpan{Byte}"/>.
         /// </summary>
         /// <param name="data">The value to be converted.</param>
         public static implicit operator ReadOnlySpan<byte>(
-            BinaryData data) =>
-            data._bytes.Span;
-
+            BinaryData data)
```

Comment:
Remember to mark the parameter nullable.

Reply:
Thanks! Will do